lua/ssr/search.lua

```diff
@@ -91,31 +91,37 @@ end
 ---@return Match[]
 function M.search(buf, node, source, ns)
   local sexpr, wildcards = build_sexpr(node, source)
-  local query = ts.parse_query(parsers.get_buf_lang(buf), sexpr)
+  local ok, query = pcall(function()
+    return ts.query.parse and ts.query.parse(parsers.get_buf_lang(buf), sexpr)
+      or ts.parse_query(parsers.get_buf_lang(buf), sexpr)
+  end)
```

Comment:
I would prefer first save parse_query to a variable, like `local parse_query = .. or ..`.